jena-text/src/main/java/org/apache/jena/query/text/TextIndexLucene.java

```diff
@@ -401,25 +402,38 @@ protected Document doc(Entity entity) {
         }
     }
 
-    private QueryParser getQueryParser(Analyzer analyzer) {
-        switch(queryParserType) {
-            case "QueryParser":
-                return new QueryParser(docDef.getPrimaryField(), analyzer) ;
-            case "AnalyzingQueryParser":
-                // AnalyzingQueryParser is deprecated in Lucene 7, switching to QueryParser
-                return new QueryParser(docDef.getPrimaryField(), analyzer) ;
-            case "ComplexPhraseQueryParser":
-                return new ComplexPhraseQueryParser(docDef.getPrimaryField(), analyzer);
-            default:
-                log.warn("Unknown query parser type '" + queryParserType + "'. Defaulting to standard QueryParser") ;
-                return new QueryParser(docDef.getPrimaryField(), analyzer) ;
-        }
-    }
-
     private Query parseQuery(String queryString, Analyzer analyzer) throws ParseException {
-        QueryParser queryParser = getQueryParser(analyzer) ;
-        queryParser.setAllowLeadingWildcard(true) ;
-        Query query = queryParser.parse(queryString) ;
+    	Query query = null;
```

Comment:
This method has a mix of tabs and spaces on each line. The Jena codebase uses spaces for indentation. Please could you change to spaces?